### PR TITLE
history-table #78

### PR DESCRIPTION
**Pull request by SarahNox**

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -24,6 +24,13 @@ var interval;
 var isTomatoOn = true;
 var almostASecondInMiliseconds = 999;
 
+if (window.webkitNotifications.checkPermission() == 0) {
+    // notfication
+} else {
+    window.webkitNotifications.requestPermission(function(){});
+}
+
+
```
Comment (sharnik) [inline app/assets/javascripts/application.js]: This is not a part of the history changes. Shouldn't be a part of this PR.

File: app/assets/javascripts/application.js [hunk 2]
```diff
@@ -57,13 +64,12 @@ function end() {
...
   document.getElementById("toggle").innerHTML = ":-)";
-  document.getElementById("toggle").disabled = true;
   document.getElementById("interrupt").disabled = true;
   clearInterval(interval);
   time = pauseTime;
-  $('.dial').val(time).trigger('configure', {'max': pauseTime , 'fgColor' : "#04B45F" });
+  $('.dial').val(time).trigger('configure', {'max': pauseTime , 'fgColor' : "04B45F" });
   setInterval(updateTimerDisplay, almostASecondInMiliseconds);
-  countdown();
+  // countdown();
```
Comment (sharnik) [inline app/assets/javascripts/application.js]: This is temporary debugging stuff, right?

File: app/views/pomodoros/_time.html.erb [hunk 3]
```diff
@@ -2,16 +2,24 @@
...
 
 <% @pomodoros.each do |pomodoro| %>
 <tr>
+  <td>
+    <% if pomodoro.success == true %>
+      <%= image_tag("tomato_middle.png") %>
+    <% else %>
+      <%= image_tag("tomato_middle_spoiled.png") %>
+    <% end %>
+  </td>
```
Comment (sharnik) [inline app/views/pomodoros/_time.html.erb]: You can shorten that into a one-liner:

image_tag( pomodoro.success? ? "tomato_middle.png" : "tomato_middle_spoiled.png" )

Or an even shorter code, but that's for extra points ;)

File: app/assets/javascripts/application.js (hunk 2 above)
Comment (SarahNox) [inline app/assets/javascripts/application.js]: yes

File: app/views/pomodoros/_time.html.erb (hunk 3 above)
Comment (SarahNox) [inline app/views/pomodoros/_time.html.erb]: nice one-liner :)

File: app/assets/javascripts/application.js (hunk 1 above)
Comment (SarahNox) [inline app/assets/javascripts/application.js]: deleted

**Pull request merged**